docs/source/reference/config.rst

```diff
@@ -68,6 +68,7 @@ Below is the configuration syntax and some example values. See detailed explanat
     :ref:`allowed_contexts <config-yaml-kubernetes-allowed-contexts>`:
       - context1
       - context2
+    :ref:`allow_all_contexts <config-yaml-kubernetes-allow-all-contexts>`: false
```

Comment:
I am thinking from a UX perspective, do we think it is possible to just make `allowed_contexts` to be available with a value `allowed_contexts: all` in addition to a list of contexts? In that case, we don't have to add another new field, i.e., keep the number of knobs lower. Wdyt?

Reply:
well, as soon as other valid values are going to be arrays, it might be OK! the patch is coming soon

Reply:
done!